lms/templates/instructor/instructor_dashboard_2/rapid_response.html

```diff
@@ -0,0 +1,28 @@
+<%page args="section_data" expression_filter="h"/>
+<%!
+from itertools import groupby
+
+from django.utils.translation import ugettext as _
+from django.urls import reverse
+
+from instructor.views.tools import get_display_name_from_usage_key
+%>
+
+
+<div>
+    % for date,runs in groupby(section_data['problem_runs'],key=lambda x:x['created'].date()):
+    <ul>
+        <li>${date.strftime('%Y/%m/%d')}</li>
+        <ul>
+            % for run in runs:
+            <li>${get_display_name_from_usage_key(run['problem_usage_key'])} - ${run['created'].strftime('%I:%m:%S %p')}:
```

Comment:
The `strftime` call needs to have a capitalized M for minutes: `'%I:%M:%S %p'`. Lowercase m is for month. Kind of unexpected, I know

Reply:
good catch :)